seedorf/games/models.py

```diff
@@ -210,14 +216,29 @@ def required_players(self):
     def get_absolute_url(self):
         return reverse("game-detail", args=[str(self.uuid)])
 
-    def get_app_link(self):
-        image = self.spot.images.first()
+    def create_share_link(self):
+        image = self.spot.images.first() if self.spot else None
```

Comment:
@Tommos0 : should we maybe put a default image to our logo if no spot image is available ?